next/src/components/_global/ctaWithBackgroundImage/ctaWithBackgroundImage.tsx

```diff
@@ -0,0 +1,21 @@
+import { type Props } from './ctaWithBackgroundImage.types';
+import styles from './ctaWithBackgroundImage.module.scss';
+import Markdown from '@/components/ui/Markdown';
+import Button from '@/components/ui/button/Button';
+
+export default function CtaWithBackgroundImage({ data: { cta, title, subtitle } }: Props) {
+  return (
+    <div className={styles.ctaWithBackgroundImage}>
+      <Markdown.h2 className={styles.title}>{title}</Markdown.h2>
+      {subtitle && <Markdown className={styles.subtitle}>{subtitle}</Markdown>}
+      {cta.text && (
+        <Button
+          data={cta}
+          svg={false}
+          theme='secondary'
```

Comment:
You doesn't declare theme from Sanity, passing the `cta` object?

Reply:
This cta is unique. User has no access to set theme in this button. The theme is set to reduce the amount of styles.